gpt4all-chat/translations/gpt4all_de_DE.ts

```diff
@@ -0,0 +1,2689 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de_DE">
+<context>
+    <name>AddCollectionView</name>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="45"/>
+        <source>← Existing Collections</source>
+        <translation>← Aktuelle Sammlungen</translation>
+    </message>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="68"/>
+        <source>Add Document Collection</source>
+        <translation>Dokumentsammlung hinzufügen</translation>
+    </message>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="78"/>
+        <source>Add a folder containing plain text files, PDFs, or Markdown. Configure additional extensions in Settings.</source>
+        <translation>Fügen Sie einen Ordner hinzu, der reine Textdateien, PDFs oder Markdown enthält. Konfigurieren Sie zusätzliche Erweiterungen in den Einstellungen.</translation>
+    </message>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="99"/>
+        <source>Name</source>
+        <translation>Name</translation>
+    </message>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="114"/>
+        <source>Collection name...</source>
+        <translation>Name diese Sammlung</translation>
+    </message>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="116"/>
+        <source>Name of the collection to add (Required)</source>
+        <translation>Name der hinzuzufügenden Sammlung (erforderlich)</translation>
+    </message>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="132"/>
+        <source>Folder</source>
+        <translation>Ordner</translation>
+    </message>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="149"/>
+        <source>Folder path...</source>
+        <translation>Ordnerpfad</translation>
+    </message>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="152"/>
+        <source>Folder path to documents (Required)</source>
+        <translation>Ordnerpfad zu Dokumenten (erforderlich)</translation>
+    </message>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="164"/>
+        <source>Browse</source>
+        <translation>Durchsuchen</translation>
+    </message>
+    <message>
+        <location filename="../qml/AddCollectionView.qml" line="177"/>
+        <source>Create Collection</source>
+        <translation>Diese Sammlung erstellen</translation>
+    </message>
+</context>
+<context>
+    <name>AddModelView</name>
+    <message>
+        <location filename="../qml/AddModelView.qml" line="55"/>
+        <source>← Existing Models</source>
```

Comment:
@cebtenzzre This will change the English base. Not sure, if this will create merge conflicts or other complications. I think it is the right word, for that location, but you are an english native speaker, so maybe you have a different opinion.